include/envoy/router/router.h

```diff
@@ -487,6 +493,11 @@ class Config {
    * (RFC1918) source.
    */
   virtual const std::list<Http::LowerCaseString>& internalOnlyHeaders() const PURE;
+
+  /**
+   * @return const std::string Gets the RouteConfiguration name.
```

Comment:
Nit: s/Gets/gets/ for consistency with above. Also, maybe use "RouteConfiguration" above.